.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: build
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+jobs:
+  build:
+    name: Build binaries
+    runs-on: ubuntu-latest
```

Comment:
Just curious, do we have other options?

Reply:
Here, https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories